### PR TITLE
[Java] Dijkstra's Shortest Path Algorithm (Adjacency Matrix)

### PR DESCRIPTION
**Pull request by sumitaccess007**
Implementation of Dijkastra's Shortest Path Algorithm in Java (Data Structure used is Adjacency Matrix)

Graph
- [x] Dijkstra's algorithm
- [ ] Edmonds–Karp algorithm
- [ ] Ford–Fulkerson algorithm
- [ ] Floyd–Warshall algorithm
- [ ] Tarjan's strongly connected components algorithm

### CODE REVIEW

File: Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
+/* Copyright 2018 Aman Mehara
```
Comment (amanmehara) [inline Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java:1]: Please write your own name and change the year to 2020.

**Review by amanmehara — CHANGES_REQUESTED** [review]

File: Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java (hunk 1 above)
Comment (sumitaccess007) [reply Java/Dijkastra Shortest Path Algorithm (Graph in form of Adjacency Matrix)/DijkastraShortestPathAlgorithm.java]: Thanks, This correction has now been taken care.

**Review by amanmehara — CHANGES_REQUESTED** [review]
Please remove the `Fibonacci Search` source file, as it is aleady merged from different PR.

Also fix the spelling of **Dijkstra**. 😛

**Comment by sumitaccess007** [comment]
> Please remove the `Fibonacci Search` source file, as it is aleady merged from different PR.
> 
> Also fix the spelling of **Dijkstra**. 😛

Hi @amanmehara 
Thanks for the Review, I have removed Fibonacci Search and Corrected Dijkstra Spelling as well.

Thanks to review again.

**Pull request merged**